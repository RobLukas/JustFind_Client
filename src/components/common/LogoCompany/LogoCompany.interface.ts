interface LogoCompanyProps {
  logoSrc: string;
}

export default LogoCompanyProps;
