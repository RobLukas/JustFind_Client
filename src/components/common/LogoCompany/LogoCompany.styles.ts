import styled from 'styled-components';

const Image = styled.img({
  position: 'absolute',
  border: 0,
  top: '50%',
  left: '50%',
  maxHeight: 40,
  maxWidth: 85,
  transform: 'translate(-50%, -50%)',
  display: 'block',
  borderRadius: 5,
});

export default { Image };
