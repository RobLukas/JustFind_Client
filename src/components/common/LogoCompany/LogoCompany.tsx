import React, { FC } from 'react';
import LogoCompanyProps from './LogoCompany.interface';
import S from './LogoCompany.styles';

const LogoCompany: FC<LogoCompanyProps> = ({ logoSrc }: LogoCompanyProps) => {
  return <S.Image src={logoSrc} />;
};

export default LogoCompany;
