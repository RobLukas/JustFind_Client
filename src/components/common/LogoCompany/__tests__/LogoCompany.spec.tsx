import React from 'react';
import { cleanup, render } from '@testing-library/react';
import 'jest-styled-components';
import LogoCompany from '../LogoCompany';

describe('<LogoCompany />', () => {
  afterEach(cleanup);
  it('renders correctly', () => {
    const urlSrc = 'urlSrc';
    const { container } = render(<LogoCompany logoSrc={urlSrc} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
