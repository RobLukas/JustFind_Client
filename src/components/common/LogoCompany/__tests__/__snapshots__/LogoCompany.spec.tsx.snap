// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoCompany /> renders correctly 1`] = `
<img
  class="sc-bdnylx bcpkxf"
  src="urlSrc"
/>
`;
